roqoqo-qryd/tests/integration/tweezer_devices.rs

```diff
@@ -438,7 +438,7 @@ fn test_number_qubits() {
         .set_tweezer_single_qubit_gate_time("PauliX", 1, 0.0, None)
         .unwrap();
 
-    assert_eq!(device.number_qubits(), 2);
+    assert_eq!(device.number_qubits(), 0);
```

Comment:
Would it be helpful to also have a test for when it is non-zero?

Reply:
If you mean testing when the qubit->tweezer mapping is updated, it's done right below it.
Now that method is the only one that influences `number_qubits()` behaviour.

Reply:
Ah I see, thank you!